src/components/SummaryCards/SummaryCard.tsx

```diff
@@ -1,12 +1,6 @@
-import React from 'react';
+import { ISummaryCardProps } from "utils/types";
 
-interface SummaryCardProps {
-  title: string;
-  value: string | number | undefined;
-  className: string;
-}
-
-const SummaryCard = ({ title, value, className } : SummaryCardProps) => {
+const SummaryCard = ({ title, value, className } : ISummaryCardProps) => {
```

Comment:
why did we removed interface from here?

Reply:
I have not remove the interface but update name by adding I in start of interface name.